internal/glfw/generate.sh

```diff
@@ -0,0 +1,16 @@
+VERSION=3.3.8
```

Comment:
Add `gen.go` including `//go:generate ./generate.sh` so that we can generate the lib by `go generate`